flow/connectors/bigquery/bigquery.go

```diff
@@ -141,6 +141,51 @@ func (bqsa *BigQueryServiceAccount) CreateStorageClient(ctx context.Context) (*s
 	return client, nil
 }
 
+// TableCheck:
+// 1. Creates a table
+// 2. Inserts one row into the table
+// 3. Deletes the table
+func TableCheck(ctx context.Context, client *bigquery.Client, dataset string) error {
+	dummyTable := "peerdb_validate_dummy"
```

Comment:
suffix a random string `_peerdb_valdiate_test_{}`

Reply:
dataset id is already randomized